### DISCUSSION TITLE
Не пускает на сервер но "Соединение установлено"

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#774] [Q&A]
Всем привет!
Возникла проблема. Не могу зайти на сервер. С чем связано не могу разобраться. Сервер скомпилировался и запускается без ошибок, но на экране клиента игры окно с надписью "Соединение установлено" и все на этом. Хотя пару дней дней назад было все норм. Помогите пожалуйста.

Error while configuring Appender GM in Logger commands.gm. Appender does not exist
AzerothCore rev. 56cd867fbe99 2024-04-06 08:17:17 +0700 (npcbots_3.3.5 branch) (Win64, Release, Static) (worldserver-daemon)

PC:
1.   Win11 Home
2.    DirectX 12.0
3.    NVIDIA GeForce RTX 3050  (8 ГБ)
4.    HexaCore Intel Core i5-12400F, 4200 MHz (42 x 100)
5.    8 ГБ DDR4-3000 DDR4 SDRAM (18-18-18-36 @ 1500 МГц) (17-18-18-36 @ 1500 МГц) (16-18-18-36 @ 1500 МГц) (15-17-17-34 
       @ 1405 МГц) (14-16-16-32 @ 1311 МГц) (13-15-15-30 @ 1218 МГц) (12-14-14-27 @ 1124 МГц) (11-13-13-25 @ 1030 МГц) (10- 
       12- 2-23 @ 937 МГц) 

boost_1_81_0-msvc-14.3-64
Win64OpenSSL-3_1_5
mysql 8

также стоит Kaspersky Premium

**Comment by trickerer** [comment]
C ошибкой `Error while configuring Appender GM in Logger commands.gm. Appender does not exist` это не связано, проверил.
Странные баги возникают иногда после обновления ядра (встречал только на АС). Попробуй полную пересборку. Build -> Clean Solution (Сборка -> Очистить Решение).

**Comment by xBarselonx** [comment]
> C ошибкой `Error while configuring Appender GM in Logger commands.gm. Appender does not exist` это не связано, проверил. Странные баги возникают иногда после обновления ядра (встречал только на АС). Попробуй полную пересборку. Build -> Clean Solution (Сборка -> Очистить Решение). Или я не об это имею ввиду))))). Простите меня пожалуйста.

делал очистить решение проекта. то увы...

**Comment by Bogir** [comment]
Буквально вчера избавился от аналогичной ситуации. После запуска в течение 2+ минут невозможно было войти на сервер. Потом сервер впускал нормально.
Ядро не пересобирал дней 5. Т.е. использовал то же, которое "лагало".
Удалил всех ботов, вычистил из world и characters и создал всех заново. 
С этого момента вход на сервер доступен в первую же секунду после запуска ядра.
При дампе нашел 89 дубликатов Entry ботов, созданных дважды в одной точке. Этих лишних удалил. Может в них была причина, или вообще в чем-то другом.
